{"uid":"7f0a07e0ddc43f4b","name":"WS-TC7 - Subscribe with invalid instrument name","fullName":"WebSocket API: book.{instrument_name}.{depth}: WS-TC7 - Subscribe with invalid instrument name","historyId":"a7879dff83f59e3b9c6360524104c121","time":{"start":1751019010623,"stop":1751019011127,"duration":504},"status":"failed","statusMessage":"AssertionError: Expected error response but none found\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/ws_steps.py\", line 134, in step_then_ws_expected\n    assert is_error_response(context), \"Expected error response but none found\"\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Expected error response but none found\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/ws_steps.py\", line 134, in step_then_ws_expected\n    assert is_error_response(context), \"Expected error response but none found\"\n","steps":[{"name":"Given WS test input \"book.INVALID_COIN.10\"","time":{"start":1751019010623,"stop":1751019011124,"duration":501},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then WS expected result should be \"error message returned\"","time":{"start":1751019011125,"stop":1751019011127,"duration":2},"status":"failed","statusMessage":"AssertionError: Expected error response but none found\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/ws_steps.py\", line 134, in step_then_ws_expected\n    assert is_error_response(context), \"Expected error response but none found\"\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"WebSocket API: book.{instrument_name}.{depth}"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"7f0a07e0ddc43f4b.json","parameterValues":[]}